pallets/ddc-payouts/src/lib.rs

```diff
@@ -1242,4 +3429,56 @@ pub mod pallet {
 			fingerprint
 		}
 	}
-}
+
+	impl<T: Config> sp_application_crypto::BoundToRuntimeAppPublic for Pallet<T> {
```

Comment:
We should leave this code inside the DDC Verification pallet. It keeps mapping between Active DAC inspectors and Validators stash keys. This Validator set is actually the Inspectors set, and access control related to Inspectors should live in DDC Verification Pallet.

We need to introduce a trait for loose coupling between the Verification and Payouts Pallet, something like `InspectorQuery` or `InspectorAuthority` that implements `is_authorized(account)` function, implement it by DDC Verification Pallet, and then use this type in DDC Payouts extrinsics' in place of `ensure!(Self::is_ocw_validator(sender.clone()), Error::<T>::Unauthorized);`

Reply:
done